### ISSUE TITLE
Only provide one Windows package: the one with JRE embedded

### ISSUE THREAD

**Issue by trnstlntk** [OpenRefine/OpenRefine#5257]
I get complaints about our Windows downloads all the time. By default, people seem to go directly to the 'Windows without JRE' version, and then are unable to start OpenRefine. They need active redirecting to the version with JRE.

### Proposed solution
Only provide one 'canonical' Windows download - which is the JRE included one, but just labeled as 'Windows version'. The 'JRE / Java' label doesn't make sense to a lot of our users anyway.

### Alternatives considered
Make the JRE version the first option, and explain more clearly (without jargon) why it's the preferred one.

### Additional context
I'm unaware of the reason/history why we have two versions to start with. Is it because we want to provide a 'lighter' version? Are there any risks or complexities attached to the JRE version?

**Comment by wetneb** [comment]
Some years ago, we only had the version without Java included. We then introduced the version with Java included precisely for the reasons you mention.
At that point we kept the version without Java for continuity, I guess. As to the reasons why the version without Java can be useful, I would list:
* if you know that you already have Java, it is lighter to download and takes less space on your machine once installed
* in some environments, it might not be allowed to bring in your own Java at all
* relying on some shared Java install makes it easier to update Java (for instance to react to vulnerabilities)

So I would be in favour of keeping distributing the version without Java, but I think it is a good idea to make it much less prominent on the download page than the version with Java. For me this should be done as part of a big overhaul of the download experience (but that's a discussion for another repository).
